app/config/services/platform.xml

```diff
@@ -226,6 +226,10 @@
         </service>
 
         <!-- Citizen projects -->
+        <service id="app.citizen_project.manager" class="AppBundle\CitizenProject\CitizenProjectManager">
```

Comment:
This service is needed ? you can use action injection (enabled in `services.xml`)

Reply:
I think this service is not conditionally used. He's used in the `return` statement : `$citizenProjectManager->getCitizenProjectAdministrators($project)`

Reply:
Yes you're right. Good catch, thanks!